tests/functional/spec/census-household.spec.js

```diff
@@ -12,7 +12,6 @@ const SelfContainedAccommodation = require('../pages/surveys/census/household/se
 const NumberOfBedrooms = require('../pages/surveys/census/household/number-of-bedrooms.page');
 const CentralHeating = require('../pages/surveys/census/household/central-heating.page');
 const OwnOrRent = require('../pages/surveys/census/household/own-or-rent.page');
-const Landlord = require('../pages/surveys/census/household/landlord.page');
```

Comment:
There's a few of these spec tests we have where they cause travis to throw an ES lint error. Might be worth getting rid of all now in this PR?